pkg/ctrlmgr/webhooks/project/project_validator.go

```diff
@@ -124,3 +131,17 @@ func (p *ProjectValidator) ValidateDelete() error {
 
 	return fmt.Errorf("the namespace %s is still exist", p.Spec.Namespace)
 }
+
+func validatorDomainSuffix(p *ProjectValidator, log clog.CubeLogger) error {
```

Comment:
better use one util func for cluster and project validator call

Reply:
just like:
func validateDomainSuffix(domains ...strings)error{
  for _, domain := range domains {
      if len(domain) == 0 {
          continue
      }
      // do you handle
  }
  return nil
}

Reply:
fixed